src/stories/ui/accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Accordion } from '../../../sections/ui/accordion/Accordion'
+
+const meta: Meta<typeof Accordion> = {
```

Comment:
Please add a little jsdoc like https://github.com/IQSS/dataverse-frontend/pull/55/commits/a073464ae0b2460ec7ec5f5873be0624b005b538

That way we get the nice docs from Storybook for free. 😄

Reply:
@ekraffmiller can you take a look at this? I think is the only thing left in the review

Reply:
yes, I added that in my latest commit